scripts/lua/sidebar.lp

```diff
@@ -77,8 +77,8 @@
                         <i class="fa fa-fw menu-icon fa-shield-alt"></i>
                         <span>Lists
                             <span class="pull-right-container">
-                                <span class="label bg-blue pull-right" id="num_lists" title="Number of subscribed and enabled lists"></span>
-                                <span class="label bg-yellow pull-right" id="num_gravity" title="Total number of domains subscribed by your Pi-hole"></span>
+                                <span class="label label-primary pull-right" id="num_lists" title="Number of subscribed and enabled lists"></span>
+                                <span class="label label-primary pull-right" id="num_gravity" title="Total number of domains subscribed by your Pi-hole"></span>
```

Comment:
```suggestion
                                <span class="label bg-yellow pull-right" id="num_gravity" title="Total number of domains subscribed by your Pi-hole"></span>
```

Reply:
it's better to use `label-warning` since it's a label component if we go with this approach.